src/app.rs

```diff
@@ -1086,9 +1441,14 @@ impl eframe::App for ProfApp {
                     window.content(ui, cx);
                 }
             }
-
             Self::cursor(ui, cx);
         });
+
+        if ctx.input().key_pressed(egui::Key::ArrowLeft) && ctx.memory().focus().is_none() {
+            ProfApp::undo_zoom(cx);
+        } else if ctx.input().key_pressed(egui::Key::ArrowRight) && ctx.memory().focus().is_none() {
+            ProfApp::redo_zoom(cx);
```

Comment:
What's `ctx.memory().focus().is_none()`? is that what avoids doing this when we're focused on some UI element?

Presumably we'll add other keybindings here, so it would be better to structure as:

```rust
if ctx.memory().focus().is_none() {
  if ctx.input().key_pressed(egui::Key::ArrowLeft) ...
}
```

I would have preferred to make the inner if statement into a match. Is that possible in egui?